src/draft-ietf-rpp-requirements.md

```diff
@@ -127,9 +127,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT","SHOULD", "SH
 
 **R4.2** Commonly used EPP extensions SHOULD be added to the RPP core data model. An example of this is the DNSSEC extension.
 
-**R4.3** RPP MUST allow an extension mechanism that allows clients to signal data omission or redaction, indicating data collected but not transmitted to the registry or redacted.
-
-A> TODO: [Issue #34](https://github.com/ietf-wg-rpp/rpp-requirements/issues/34)
```

Comment:
Please leave the reference as commented out.
See https://github.com/ietf-wg-rpp/rpp-requirements/pull/128/files#diff-4767e173ad76477b2ca5dc5b558164d51556da4982bdf225d1e9b9d87cd1a2e2R162